### PR TITLE
DOMA-3565 Added MeterReadingSource type for external import

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

File: apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// auto generated by kmigrator
+// KMIGRATOR:0150_alter_meterreadingsource_type:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMC4yIG9uIDIwMjItMDctMjAgMDY6MDUKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAxNDlfdGlja2V0X2NvbXBsZXRlZGF0X3RpY2tldGNoYW5nZV9jb21wbGV0ZWRhdGZyb21fYW5kX21vcmUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWx0ZXJGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nbWV0ZXJyZWFkaW5nc291cmNlJywKICAgICAgICAgICAgbmFtZT0ndHlwZScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5DaGFyRmllbGQoY2hvaWNlcz1bKCdpbXBvcnRfY29uZG8nLCAnaW1wb3J0X2NvbmRvJyksICgnY2FsbCcsICdjYWxsJyksICgnbW9iaWxlX2FwcCcsICdtb2JpbGVfYXBwJyksICgnZXh0ZXJuYWxfaW1wb3J0JywgJ2V4dGVybmFsX2ltcG9ydCcpXSwgbWF4X2xlbmd0aD01MCksCiAgICAgICAgKSwKICAgIF0K
+
+exports.up = async (knex) => {
```
Comment (akarjakin) [inline apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js:4]: can we get rid of this useless migration?

**Review by akarjakin — APPROVED** [review]

File: apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js:4]: Should we? Since every developer will get locally btw.

Comment (pahaz) [reply apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js:4]: it's imposible. But we can move the second line to the previous migration. to avoin a new file creation

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/meter/constants/constants.js [hunk 2]
```diff
@@ -11,13 +11,15 @@ const GAS_SUPPLY_METER_RESOURCE_ID = '1c267e92-0631-11ec-9a03-0242ac130003'
 const METER_MODAL_FORM_ITEM_SPAN = 11
 
 const METER_READING_SOURCE_CALL_TYPE = 'call'
-const METER_READING_SOURCE_IMPORT_TYPE = 'import_condo'
+const METER_READING_SOURCE_INTERNAL_IMPORT_TYPE = 'import_condo'
+const METER_READING_SOURCE_EXTERNAL_IMPORT_TYPE = 'external_import'
```
Comment (pahaz) [inline apps/condo/domains/meter/constants/constants.js:15]: external system?

**Review by AntonAL — APPROVED** [review]

File: apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js (hunk 1 above)
Comment (SavelevMatthew) [reply apps/condo/migrations/20220720110510-0150_alter_meterreadingsource_type.js]: merged with previous 👍

**Pull request merged**